JSONRenderer.php

```diff
@@ -44,6 +47,14 @@ public function renderReport(Report $report)
                 );
             }
 
+            $name = $violation->getMethodName() ?: $violation->getFunctionName() ?: $violation->getClassName();
```

Comment:
Scratch that - that's only available in PHP 7 which may not be available on alpine. Punting for now